study-builder/studies/rarex/study.conf

```diff
@@ -633,65 +708,65 @@
           ],
           "files": [
             {
-              "filepath": "assets/consent_lar.pdf",
+              "filepath": "pdfs/lar-icf.pdf",
               "type": "CUSTOM",
               "substitutions": [
                 {
                   "type": "BOOLEAN",
                   "field": "follow_up_yes",
-                  "activityCode": "LAR_CONSENT",
+                  "activityCode": "LAR_CONSENT_ASSENT",
```

Comment:
This PDF configurations pulls data from `LAR_CONSENT` activity (see `sources` above). Why are these substitutions pointing at `LAR_CONSENT_ASSENT`?

Reply:
That's true! fixing